app/controllers/events_controller.rb

```diff
@@ -204,4 +237,46 @@ def rss
       format.rss { render layout: false }
     end
   end
+
+  private
+
+  def display_all_parameter_options(events)
```

Comment:
The idea behind using this method was to only show options that were valid, e.g. if event category "DJ" is chosen by the user, then only areas that have been used at a "DJ" event should be available in the dropdown and vice versa.  

But the overhead for this kind of functionality might be too high.